src/modules/esm.mjs

```diff
@@ -0,0 +1,42 @@
+import path from "path";
+import fs from "fs/promises";
+import { fileURLToPath } from "url";
+import { release, version } from "os";
+import { createServer as createServerHttp } from "http";
+import * as c from "./files/c.js";
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const filePathA = path.join(__dirname, "files/a.json");
+const filePathB = path.join(__dirname, "files/b.json");
+
+const random = Math.random();
+
+let unknownObject;
+
+if (random > 0.5) {
+  unknownObject = JSON.parse(await fs.readFile(filePathA));
```

Comment:
If no encoding is specified (using options.encoding), the data is returned as a [<Buffer>](https://nodejs.org/api/buffer.html#class-buffer) object. Otherwise, the data will be a string.